docs/DEV_GUIDE.md

```diff
@@ -405,6 +401,7 @@ For example, in this case we will add messages for the `required` and the `patte
 // inject the NgxFormErrorsMessageService to define the messages
 
 formErrorsMessageService.addErrorMessages({
+	"foo.required": "The foo field is obviously required!",
```

Comment:
This is what I'm concerned about: I'm not sure whether the `.` syntax should be used for everything (errorGroup and formControlName) because then it gets confusing for the developer. What if he has an errorGroup and a formControlName with exactly the same name?